src/amo/components/Addon/index.js

```diff
@@ -177,12 +184,24 @@ export class AddonBase extends React.Component {
       isPreviewingTheme,
       installStatus,
     } = this.props;
-    const previewURL = addon ? addon.previewURL : null;
     const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
-    const iconUrl = getAddonIconUrl(addon);
 
-    if (type === ADDON_TYPE_THEME) {
-      const label = isPreviewingTheme ? i18n.gettext('Cancel preview') : i18n.gettext('Tap to preview');
+    // The 'tap to preview' button will be fully removed in the following PR:
+    // https://github.com/mozilla/addons-frontend/pull/4914
+    // for now this is only displayed over lightweight preview images
+    if (this.addonIsTheme()) {
+      let previewURL = addon ? addon.previews.length > 0 && addon.previews[0].image_url : null;
```

Comment:
I don't think you need to test `addon` here since `addonIsTheme()` does it already.

Also, I find the mix of ternary and logical operator very difficult to read.